### ISSUE TITLE
Interface/Send messages from non-Android device?

### ISSUE THREAD

**Issue by rflmota** [ibrdtn/android-whisper#7]
Hello,

I'd like to know if is it possible to interface, send/receive messages, with this chat from other devices other than Android? What's the endpoint and message formats?

**Comment by morgenroth** [comment]
Yes, it is possible. Chat messages are just plain text. The endpoint is simply "chat".

You can use dtnsend on a desktop like this.
```
echo "Hello" | dtnsend dtn://android-xxxx.dtn/chat
```

**Comment by rflmota** [comment]
Thank you a lot @morgenroth for your support, really appreciated.

I'd like to know if its possible also to broadcast messages to all users? Is there any global endpoint?
How about the presence/status message? Does it have a custom format?

**Comment by morgenroth** [comment]
A broadcast for chat messages doesn't exists.
Presence messages are broadcasted. They are plain text and not hard to understand. Give it a try and subscribe to dtn://chat.dtn/presence.

**Comment by mfzn** [comment]
sorry for necroposting, but is it possible to send presence from a pc(lets say raspberry pi) to android device?
